mead/config/sst2-bert-hf-base-uncased.json

```diff
@@ -27,7 +27,7 @@
     "model_type": "fine-tune"
   },
   "train": {
-    "epochs": 5,
+    "epochs": 1,
```

Comment:
Did you mean to change number of epochs?  If you want to test this, you can pass

`mead-train --x:train.epochs 1 ...`

Reply:
oh i didnt.

Reply:
updated